src/main/resources/script.sql

```diff
@@ -0,0 +1,13 @@
+DROP DATABASE IF EXISTS ridemart;
+CREATE DATABASE ridemart;
+USE ridemart;
+
+CREATE TABLE user (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    username VARCHAR(255) NOT NULL UNIQUE,
```

Comment:
you dont need 255 characters for any of your columns
please, update the size of the columns